sdk/keyvault/keyvault-admin/recordings/node/keyvaultbackupclient/recording_beginbackup.js

```diff
@@ -29,7 +29,7 @@ nock('https://eastus2.keyvault_name.managedhsm-int.azure-int.net:443', {"encoded
   'no-cache' ]);
 
 nock('https://eastus2.keyvault_name.managedhsm-int.azure-int.net:443', {"encodedQueryParams":true})
-  .post('/backup', {"storageResourceUri":"https://chrissprim.blob.core.windows.net/backup","token":"?sv=2019-12-12&ss=bfqt&srt=sco&sp=rwdlacupx&se=2022-07-14T05:56:52Z&st=2020-09-03T21:56:52Z&spr=https,http&sig=Rzts1hBG%2BgqTCTZo%2BX%2FZgJWJ5Ao16ueN%2F1BB4Dg8%2FLo%3D"})
```

Comment:
The keys of these SAS tokens were revoked within the same day of the recording. They're harmless. The team is aware.

Reply:
This PR now includes a fix that prevents further SAS tokens to be leaked for the kv-admin.